blog/build-a-network-speed-monitor-in-bash-with-awk.md

```diff
@@ -0,0 +1,110 @@
+# Build a network speed monitor in bash with AWK
+
+Sometimes we need to measure the network speed of a server but some services like `speedtest` are not useful because we want to check our bandwidth with another cloud service and not the internet itself.
```

Comment:
```suggestion
Sometimes we need to measure the network speed of a server, but services like `speedtest` are not useful because we want to check our bandwidth against another cloud service and not the internet itself.
```